src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int MAX_BALLS = 3;
+    private static final int START_INDEX = 0;
```

Comment:
It's pretty expected that array is starting from 0 index, so you please don't worry about it